dask/dask-schema.yaml

```diff
@@ -147,3 +147,15 @@ properties:
               comprehensible, but it comes at the cost of additional processing. If
               False, then the top-most key will be used. For advanced usage, a function
               to create the new name is also accepted.
+
+          blockwise-annotations:
```

Comment:
(More of a thought/comment): I definitely agree that we want this to be configurable. Making this a boolean option seems fine for now, but I can imagine a world in which the user can specify *which* of the rules they want to agree to (rather than all-or-none). I'm thinking "True" or "False" should still work in that case, so perhaps we wouldn't need to "break" this initial design.